src/shared/tpm2-util.c

```diff
@@ -133,6 +133,85 @@ static inline void Esys_Freep(void *p) {
                 sym_Esys_Free(*(void**) p);
 }
 
+/* Get a specific TPM capability (or capabilities).
+ *
+ * If successful, returns 0 if there are more available properties, even if the requested count was reached,
+ * or 1 if there are no more available properties. This corresponds to the TPM2_GetCapability() "more"
+ * parameter; specifically, this returns 0 if "more" is TPM2_YES (true), and 1 if "more" is TPM2_NO
+ * (false). */
```

Comment:
hmm, maybe invert this? i.e. EOF is indicated via read() returning 0, maybe we should follow this same logic here. i.e. return > 0 if there's more to come, and == 0 if not and we reached the end.

i think this would be less surprising.

Reply:
i switched the return 0/1 behavior and tried to clarify the comment on the behavior of the function.